tasks/agent.py

```diff
@@ -434,6 +440,8 @@ def clean(ctx):
     print("Remove agent binary folder")
     ctx.run("rm -rf ./bin/agent")
 
+    print("Cleaning rtloader")
+    rtloader_clean(ctx)
```

Comment:
Should we only clean when `exclude_rtloader` (or `puppy`) is not set, to make the build behave exactly the same as before when `exclude_rtloader` / `puppy` is set?

Reply:
We do not have the arguments `exclude_rtloader` (or `puppy`) in `clean` function and I do no think it worth adding them.